src/main/steps/applicant/yourdocuments/alldocuments/alldocuments/tasklist-items-all-documents.ts

```diff
@@ -19,12 +19,12 @@ export const applicant_tasklist_items_all_docs_en = {
   drug_alcohol_tests: 'Drug and alcohol tests (toxicology)',
   police_disclosures: 'Police reports',
   witness_availability: 'Witness availability',
-  tenancy_and_mortgage_availability: 'Tenancy and mortgage',
-  // respondent_response_to_request_for_child_arrangements:
-  //   "<namerespondentxxxxx>'s response to the request for child arrangements",
+  tenancy_and_mortgage_availability: 'Tenancy and mortgage agreements',
```

Comment:
Where are the necessary translations for welsh?

Reply:
added welsh tags